docs/docs/installation_and_setup/install.md

```diff
@@ -28,26 +28,31 @@ for parallelization, which is automatically installed as part of the
 `pip` and `conda` install commands above.
 
 
-## How to Install Conda {#conda}
+### How to Install Conda {#conda}
 
-Install conda using the instructions below.
-
-### On Linux
+You can install [Miniforge](https://github.com/conda-forge/miniforge),
+[Anaconda](https://www.anaconda.com/download), or other distributions of Conda. Miniforge can be installed easily
+on Linux, MacOS and WSL using terminal commands:
 
 ```shell
-wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-chmod +x miniconda.sh
-./miniconda.sh -b
-export PATH=$HOME/miniconda3/bin:$PATH
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+bash Miniforge3-$(uname)-$(uname -m).sh
```

Comment:
This is missing installing Bodo step to match Windows WSL section.
Need a link to follow steps in the previous section

Reply:
This section says "How to Install Conda", and I made it a subsection of the previous section. Conda installation is right above it.